helm_deploy/values-prod.yaml

```diff
@@ -1,40 +1,20 @@
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
+---
+# Environment specific values, override helm_deploy/hmpps-manage-users/values.yaml
 
-replicaCount: 4
+generic-service:
+  ingress:
+    host: manage-users.hmpps.service.justice.gov.uk
 
-image:
-  repository: quay.io/hmpps/hmpps-manage-users
-  tag: latest
-  pullPolicy: IfNotPresent
-  port: 3000
+  env:
+    API_ENDPOINT_URL: https://api.prison.service.justice.gov.uk
+    MANAGE_USERS_API_ENDPOINT_URL: https://hmpps-manage-users-api.hmpps.service.justice.gov.uk
+    OAUTH_ENDPOINT_URL: https://sign-in.hmpps.service.justice.gov.uk/auth
+    DPS_ENDPOINT_URL: https://digital.prison.service.justice.gov.uk
+    HMPPS_COOKIE_NAME: hmpps-session-prod
+    WEB_SESSION_TIMEOUT_IN_MINUTES: "60"
```

Comment:
this could have gone in the values.yaml too

Reply:
you are right - will make a note and push back with next PR